r7insight/utils.py

```diff
@@ -139,17 +141,14 @@ def run(self):
     class TLSSocketAppender(PlainTextSocketAppender):
         def open_connection(self):
             sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
-            sock = ssl.wrap_socket(
-                sock=sock,
-                keyfile=None,
-                certfile=None,
-                server_side=False,
-                cert_reqs=ssl.CERT_REQUIRED,
-                ssl_version=ssl.PROTOCOL_TLSv1_2,
-                ca_certs=certifi.where(),
-                do_handshake_on_connect=True,
-                suppress_ragged_eofs=True,
-            )
+
+            # Create an SSL context
+            context = ssl.SSLContext(ssl.PROTOCOL_TLS_CLIENT)
```

Comment:
only functionality change: allows for client to negotiate best TLS version (1.3 potentially)

whereas previous code enforced TLS1.2

Reply:
Does this break compatibility with any older versions?

Reply:
depending on what you're referring to:
- regarding TLS backwards-compatability: iiuc won't break anything since it'll get the latest supported TLS version
- regarding python backwards-compatability: `SSLContext.wrap_socket()` has existed since Python 2.7/3.2, so don't think so